### ISSUE TITLE
Evaluate Figma's accessibility features per request of the design leads

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#57911]
## What does your team need support for? Check all that apply.

- [ ] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [x] Something else 

OCTO design leads are investigating what would be the impacts of migrating from Sketch to Figma. Does it have accessibility features we'd like to take advantage of? We'd like a CAIA expert to dive into Figma and find out for sure. 

Here is a sample Figma file that they provide to describe their accessibility features.

https://www.figma.com/file/Di4aGuIQ8JvoG7oEbyjYiw/Accessible-Prototypes-Playground-(Community)?node-id=7-2

## Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

## When does this work need to be done?
- Estimated launch date: n/a
- Estimated staging review date: n/a
- Accessibility needed by:  Unsure - check with @naomimarcussen who is the requester

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [x] No, let's work asynchronously and meet if needed

## About your team

OCTO-DE product owner: @naomimarcussen @artsymartha68

**Comment by sara-amanda** [comment]
## Items to review

- [Accessible Prototypes in Figma](https://help.figma.com/hc/en-us/articles/7810391964695-Accessible-prototypes-in-Figma#:~:text=Accessible%20prototypes%20in%20Figma%20allow,install%20a%20supported%20screen%20reader.)
- [Accessibility Tools for Figma](https://www.digitala11y.com/accessibility-tools-and-resources-for-designers/#accessibility-tools-for-figma)
- [Accessibility Tools for Sketch](https://www.digitala11y.com/accessibility-tools-and-resources-for-designers/#accessibility-tools-for-sketch)
- [Figma Plugin List](https://www.figma.com/community/tag/accessibility/plugins)

**Comment by sara-amanda** [comment]
## CAIA Feedback

[The linked Google Doc for the Sketch to Figma comparison has been updated](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit?usp=sharing) with collective feedback from the Mobile UX Team c/o @drjecker (@brea11y). (Thank you, Jen and Brea!)

- Additional feedback will be added as it is received. 
- Feedback status is also reflected in the ticket body.

### CAIA Tasks
- [x] Comparison table ([general](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.en5abt30vve6)) Sketch vs. Figma
- [x] Comparison table ([accessibility](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.20axupp9pdf)) Sketch vs. Figma
- [ ] [VA Designer Feedback on Figma vs. Sketch](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.ac05d1r2ectq) **(In Progress)**

## CAIA Recommendation

- Initial recommendation is trending towards Figma, at the moment, as of 5/23/2023 at 8:56 p.m. EST
- This recommendation is subject to change, as it is dependent on additional feedback.

@naomimarcussen @artsymartha68 @coforma-terry @SarahKay8

**Comment by sara-amanda** [comment]
## CAIA Feedback

Please find the following linked [Google Doc, as a starting point for the Sketch to Figma comparison](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit?usp=sharing). Feedback was added as it was received. 

### CAIA Tasks
- [x] Comparison table ([general](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.en5abt30vve6)) Sketch vs. Figma
- [x] Comparison table ([accessibility](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.20axupp9pdf)) Sketch vs. Figma
- [x] [VA Designer Feedback on Figma vs. Sketch](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit#heading=h.ac05d1r2ectq) **(Completed)**

## CAIA Recommendation

- Initial recommendation is trending towards Figma, at the moment, as of 5/12/2023 at 12:47 p.m. EST
- **This recommendation is complete, with the response window being open for one month, and including all but two, who did not respond.** 
    - **Initial outreach**: 5/11/2023
    - **Final responses received**: 6/12/2023
    - **Responses received from**: 5 individuals + 1 team's response
- ⭐ **_The overall preference with designers is Figma._** 

## Figma References

- [x] Brea Blackwelder, VA Mobile App (Figma only)
- [x] Jen Ecker PLUS the Mobile UX Team, VA Mobile App (Sketch to Figma)
- [ ] Lauren Russell, VA Mobile App (Sketch to Figma)
- [ ] Holly Collier, VA Mobile App (Sketch to Figma)
- [x] Jordan Wood, Public Websites Team (Sketch to Figma)
- [x] Andrew Bergeron, VA.gov Forms Team (Figma experience and preference)
- [x] Ciera Maddox, VAOS Team (Sketch to Figma) Trying out some beta prototyping with screen reader functionality

**Comment by sara-amanda** [comment]
@RLHecht @coforma-terry  this is ready to close, as I have moved it to "validate." Thanks!

**Comment by RLHecht** [comment]
Thanks! Closing!

**Comment by sara-amanda** [comment]
## CAIA UPDATE ON ACCESSIBLE PROTOTYPES

- [x] [Documentation updated 7/27/2023](https://docs.google.com/document/d/1HTOMlFttclwi9k2Cf-aKU-Q690-o-m4jm7xiKnYhT5c/edit?usp=sharing)
- [x] [Ref. ticket #62359](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62359#issuecomment-1652769970) Core Identity Team (upcoming Research in August 2023)
- [x] Reached out to Angela Fowler, and she said she would be interested in testing this when available.
<hr>

### Accessible Prototypes in Figma (Desktop)
- While in `accessibility mode`, Figma design elements are translated to HTML.
    - **Screen readers** interpret HTML tags to understand the content and sections of a document, as well as which elements are available for a user to interact with on a page.
    - **Semantic HTML tags** also allow a screen reader to communicate content that may be highlighted visually to a user.
    - [Learn more about Figma's Accessible Prototypes](https://help.figma.com/hc/en-us/articles/7810391964695-Accessible-prototypes-in-Figma)
- **However, this is for non-mobile prototypes only** - hopefully, mobile is not too far behind in beta. 
    - Accessible prototypes are only available in the desktop browser and on the Figma desktop app. 
    - Accessibility mode does not work for mobile or mobile web prototypes. 
    - [Learn more about browser and OS requirements](https://help.figma.com/hc/en-us/articles/360039827194-Figma-browser-requirements). 

- Words of caution from a Figma [Accessible Prototype User(ref. article)](https://bootcamp.uxdesign.cc/i-tried-the-prototype-screen-reader-in-figma-beta-ceda511d715a) 
    - Build your prototype with HTML in mind. 
    - Evaluate your user flow carefully. 
    - Try it yourself.

cc: @naomimarcussen @artsymartha68
